README.md

```diff
@@ -89,24 +91,98 @@ const ckanPackage = frictionlessCkanMapper.packageFrictionlessToCkan(
 )
 
 console.log(ckanPackage)
-// {
-//     name: 'gdp',
-//     notes: 'Country, regional and world GDP in current USD.',
-//     url: 'https://datopian.com',
-//     title: 'Countries GDP',
-//     version: '1.0',
-//     extras: [
-//       { key: 'newdict', value: [Object] },
-//       { key: 'newint', value: 123 }
-//     ],
-//     resources: [
-//       {
-//         name: 'data.csv',
-//         url: 'http://someplace.com/data.csv',
-//         size: 100
-//       }
-//     ]
-//   }
+/*
+{
+    name: 'gdp',
```

Comment:
I think this may be a bit of overkill in README but not absolutely sure. I don't think we need output listed too much.

Reply:
Agree